posthog/celery.py

```diff
@@ -42,6 +43,15 @@ def setup_periodic_tasks(sender, **kwargs):
     sender.add_periodic_task(15 * 60, calculate_cohort.s(), name="debug")
     sender.add_periodic_task(600, check_cached_items.s(), name="check dashboard items")
 
+    action_event_mapping_interval_minutes = settings.ACTION_EVENT_MAPPING_INTERVAL_MINUTES
```

Comment:
I don't think this should be an env variable. Let's just pick a sensible default. Every 10 minutes seems fine to me